R/backend-postgres-old.R

```diff
@@ -15,17 +15,19 @@ db_write_table.PostgreSQLConnection <- function(con,
                                                 values,
                                                 temporary = TRUE,
                                                 ...) {
-
+  table <- as_table_ident(table)
   if (!isFALSE(temporary)) {
     cli_abort(c(
       "RPostgreSQL backend does not support creation of temporary tables",
       i = "Either set {.code temporary = FALSE} or switch to {.pkg RPostgres}"
     ))
   }
 
+  # RPostgreSQL doesn't handle `Id()` or `SQL()` correctly, so we can only pass
```

Comment:
Do we want to warn about this? Or do we think so few people still use RPostgreSQL that it's not worth it?